packages/grafana-ui/src/components/Slider/styles.ts

```diff
@@ -14,7 +14,8 @@ export const getStyles = stylesFactory((theme: GrafanaTheme2, isHorizontal: bool
   return {
     container: css`
       width: 100%;
-      margin: ${isHorizontal ? 'none' : `${spacing(1, 3, 1, 1)}`};
+      margin: ${isHorizontal ? 'inherit' : `${spacing(1, 3, 1, 1)}`};
+      padding-bottom: ${isHorizontal ? theme.spacing(1) : 'inherit'};
```

Comment:
This creates too much bottom padding for normal sliders 
![Screenshot from 2021-11-10 16-11-10](https://user-images.githubusercontent.com/10999/141139091-c36bc9f8-a88a-4135-8d49-fec2b958ebc5.png)

Reply:
Changed it so that the padding is only added when marks have been defined.